LEAF_Request_Portal/Email.php

```diff
@@ -109,9 +109,9 @@ public function clearRecipients(): void
 
     /**
      * Set email sender object variable
-     * @param $strAddress
+     * @param string|null $strAddress
      */
-    public function setSender(string $strAddress): void
+    public function setSender(string|null $strAddress): void
     {
```

Comment:
Not that I want you to change it now, but this could be written as ?string $strAddress. the ? says that it can  be nullñ

Reply:
Oh okay, did not know there was a shorthand way of writing that. I will look into that.